src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
+      "S22": { "title": "Algo President", "tier": 11 },
+      "F22": { "title": "Algo Team Lead", "tier": 11 }
+    }
   },
   {
     "fullName": "Jason Anthony",
-    "positions": {
-      "S21": {
-        "title": "Secretary",
-        "tier": 3
-      }
-    },
-    "picture": "jason-anthony.webp"
+    "picture": "jason-anthony.webp",
+    "positions": { "S21": { "title": "Secretary", "tier": 3 } }
   },
   {
     "fullName": "Angel Armendariz",
-    "positions": {
-      "S22": {
-        "title": "Dev Project Manager",
-        "tier": 22
-      }
-    },
-    "picture": "angel-armendariz.webp"
+    "picture": "angel-armendariz.webp",
+    "positions": { "S22": { "title": "Dev Project Manager", "tier": 18 } }
   },
   {
     "fullName": "Sami Bajwa",
+    "picture": "sami-bajwa.webp",
     "positions": {
-      "F21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      },
-      "S22": {
-        "title": "Algo Officer, nodebuds Officer",
-        "tier": 15
-      },
-      "F22": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "sami-bajwa.webp"
+      "F21": { "title": "nodebuds Officer", "tier": 22 },
+      "S22": { "title": "Algo Officer, nodebuds Officer", "tier": 22 },
+      "F22": { "title": "Algo Officer", "tier": 12 }
+    }
   },
   {
     "fullName": "Johnathan Carranza",
-    "positions": {
-      "F21": {
-        "title": "Dev Officer",
-        "tier": 21
-      }
-    },
-    "picture": "johnathan-carranza.webp"
+    "picture": "johnathan-carranza.webp",
+    "positions": { "F21": { "title": "Dev Officer", "tier": 19 } }
   },
   {
     "fullName": "Alan Cortez",
-    "socials": {
-      "website": "alancortez1337.github.io/portfolio/",
-      "github": "alancortez1337"
-    },
+    "picture": "alan-cortez.webp",
+    "socials": { "website": "alancortez1337.github.io/portfolio/", "github": "alancortez1337" },
     "positions": {
-      "S22": {
-        "title": "Create Officer",
-        "tier": 17
-      },
-      "F22": {
-        "title": "Design Team Lead",
-        "tier": 17
-      }
-    },
-    "picture": "alan-cortez.webp"
+      "S22": { "title": "Create Officer", "tier": 16 },
+      "F22": { "title": "Design Team Lead", "tier": 13 }
+    }
   },
   {
     "fullName": "Wesley Chou",
+    "picture": "wesley-chou.webp",
     "positions": {
-      "S21": {
-        "title": "Dev Officer",
-        "tier": 21
-      },
-      "F21": {
-        "title": "Dev Director",
-        "tier": 20
-      },
-      "S22": {
-        "title": "Dev President",
-        "tier": 20
-      },
-      "F22": {
-        "title": "Marketing Co-Lead",
-        "tier": 23
-      }
-    },
-    "picture": "wesley-chou.webp"
+      "S21": { "title": "Dev Officer", "tier": 19 },
+      "F21": { "title": "Dev Director", "tier": 17 },
+      "S22": { "title": "Dev President", "tier": 17 },
+      "F22": { "title": "Marketing Co-Lead", "tier": 8 }
```

Comment:
The updated "tier" for Marketing Leader is '9'

`"Marketing Leader": { "id": 9, "index": 550 }`